vm/actor/src/builtin/power/types.rs

```diff
@@ -16,34 +16,25 @@ pub type SectorTermination = i64;
 pub const SECTOR_TERMINATION_EXPIRED: SectorTermination = 0;
 /// Unscheduled explicit termination by the miner
 pub const SECTOR_TERMINATION_MANUAL: SectorTermination = 1;
+/// Implicit termination due to unrecovered fault
+pub const SECTOR_TERMINATION_FAULTY: SectorTermination = 3;
```

Comment:
no need to change now, just thinking out loud this should probably be an enum in the future